src/olympia/amo/cron.py

```diff
@@ -159,28 +159,6 @@ def collection_subscribers():
 
 
 @cronjobs.register
-def unconfirmed():
```

Comment:
This wasn't running. I got Wei to run some queries on prod for me and there are 391,260 users with a confirmation code and 390,592 users that match this query. Perhaps we should delete those users before we drop `confirmationcode`?